server/src/main/java/io/spine/server/projection/ProjectionRepository.java

```diff
@@ -273,7 +273,7 @@ public Timestamp readLastHandledEventTime() {
     EventStreamQuery createStreamQuery() {
         Set<EventFilter> eventFilters = createEventFilters();
 
-        // Get the timestamp of the last event. This also ensures we have the storage.
+        // Gets the timestamp of the last event. This also ensures we have the storage.
```

Comment:
This is not Javadoc of a method. It deliberately starts with imperative to convey the meaning of the following line.